.github/workflows/snapshot.yml

```diff
@@ -23,6 +23,11 @@ jobs:
           metadata-file-path: '.github/project.yml'
           local-file: 'true'
 
+      - name: Set Image Tag Env
+        run: |
+          echo "PROJECT_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
+          echo "NEXT_VERSION=$(echo ${{steps.metadata.outputs.next-version}} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
```

Comment:
These should both be the same value, maybe using just `NEXT_VERSION` shows more clearly that the snapshot is for "next".